doc/redirects.md

```diff
@@ -0,0 +1,53 @@
+# Redirects
+
+Sometimes (docs) pages get removed, renamed, and added. There is [a redirects file](../www/_data/redirects.yml) that contains redirects for such occasions. It has three sections:
+
+- `general`: single-page redirects.
+- `docs-global`: redirects across all docs versions and languages,
+- `docs`: version-specific docs redirects, and
```

Comment:
The items in this list are in the wrong order.

Reply:
Hm, strange how this happened, these are indeed different than in the original.
But why is this "wrong", what defines the order?

Reply:
Just the commas and the "and". I'm just pointing out that the writing is currently wrong, so it just needs to be fixed to match, or the order should be back to what it was before. The actual order doesn't matter.

Reply:
Oh, right. Fixed.